python/sbp/client/loggers/json_logger.py

```diff
@@ -33,9 +33,9 @@ def fmt_msg(self, msg):
       data = self.dispatcher(msg).to_json_dict()
     except KeyError:
       data = msg.to_json_dict()
-    return {"delta": self.delta(),
-            "timestamp": self.timestamp(),
-            "data": data}
+    msg = {"delta": self.delta(), "timestamp": self.timestamp(), "data": data}
```

Comment:
Yeah, I actually reversed it - but then thought that might be desirable to overwrite. But probably not.